src/coreclr/vm/ecalllist.h

```diff
@@ -1107,6 +1109,9 @@ FCClassElement("AssemblyExtensions", "System.Reflection.Metadata", gAssemblyExte
 FCClassElement("AssemblyLoadContext", "System.Runtime.Loader", gAssemblyLoadContextFuncs)
 
 FCClassElement("AssemblyName", "System.Reflection", gAssemblyNameFuncs)
+#ifdef FEATURE_OBJCWRAPPERS
```

Comment:
Do we want to keep the name `FEATURE_OBJCWRAPPERS` now that we've renamed the API?

Reply:
`FEATURE_OBJCBRIDGE`?

Reply:
I'll do a follow-up change of renaming the define and deleting the current `Wrappers` class (and friends).

Reply:
I like that name.